### PR TITLE
Added in xml syntax check

### PR DESCRIPTION
**Pull request by bam95899**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #Pycharm
 .idea
-
+.DS_Store
```
Comment (phg49389) [inline .gitignore:3]: Why did you remove the .DS_Store line from .gitignore?

File: sitemap_cron.py [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+from xml.sax import make_parser
 
 
 client = Client(config.SENTRY_URL)
 
 
+def parse_file(filename):
+    parser = make_parser()
+    parser.setContentHandler(ContentHandler())
+    parser.parse(filename)
```
Comment (phg49389) [inline sitemap_cron.py:20]: Does this method need to have a return value?

**Review by phg49389 — COMMENTED** [review]
Just 2 questions for you

File: .gitignore (hunk 1 above)
Comment (bam95899) [reply .gitignore:3]: I added .DS_Store to .gitignore, not removed it.

File: sitemap_cron.py (hunk 2 above)
Comment (bam95899) [reply sitemap_cron.py:20]: The method is just used to check if the syntax is correct, so it doesn't need to return anything. If the method fails, then an exception is caught and sent out via sentry and the program stops.

File: .gitignore (hunk 1 above)
Comment (phg49389) [reply .gitignore:3]: Ah, yes. I am blind. Good point!

File: sitemap_cron.py (hunk 2 above)
Comment (phg49389) [reply sitemap_cron.py:20]: 👍

**Review by phg49389 — APPROVED** [review]

File: sitemap_cron.py (hunk 2 above)
Comment (c-schwarze) [reply sitemap_cron.py:20]: Might be worth a comment mentioning that

**Review by c-schwarze — APPROVED** [review]

File: sitemap_cron.py (hunk 2 above)
Comment (bam95899) [reply sitemap_cron.py:20]: Added a comment!
